nengo_dl/tests/test_builder.py

```diff
@@ -48,29 +47,31 @@ def build_step(self, signals):
 
                 return 0, 1
 
-            def build_post(self, ops, signals, config):
+            def build_post(self, signals):
                 self.post_built = True
 
-    builder.build_pre(progress)
+    builder = Builder([ops])
 
-    result = builder.build(progress)
+    builder.build_pre(None, None, progress)
```

Comment:
Maybe use keyword args just so it's clear what each one is (and if arguments change again it might be easier to catch it).